File: 프로그래머스/0/181930. 주사위 게임 2/주사위 게임 2.java
class Solution {
    public int solution(int a, int b, int c) {
        if (a == b && b == c) {
            return (a + b + c) * (a * a + b * b + c * c) * (a * a * a + b * b * b + c * c * c);
        } else if ((a == b && b != c) || (a != b && b == c) || (a == c && a != b)) {
            return (a + b + c) * (a * a + b * b + c * c);
        } else {
            return a + b + c;
        }
    }
}

File: 프로그래머스/0/181930. 주사위 게임 2/주사위 게임 2.kt
class Solution {
    fun solution(a: Int, b: Int, c: Int): Int {
        return if(a == b && b == c){
            (a + b + c) * (a*a + b*b + c*c) * (a*a*a + b*b*b + c*c*c)
        } else if ((a == b && b != c) || (a != b && b == c) || (a == c && a != b)) {
            (a + b + c) * (a*a + b*b + c*c)
        } else {
            a + b + c
        }
    }
}